userspace/libsinsp/parsers.cpp

```diff
@@ -1820,7 +1820,7 @@ void sinsp_parser::schedule_mesos_events(sinsp_evt *evt)
 		int event_count = mesos_client->get_capture_events().size();
 		if(event_count)
 		{
-			m_mesos_metaevents_state.m_piscapevt->tid = evt->get_tid();
+			m_mesos_metaevents_state.m_piscapevt->tid = 0;
```

Comment:
Why did you remove this (and the others similar)?

Reply:
It allowed me to separate the functions for k8s/mesos meta event generation, which were really using m_metaevt, from m_meta_evt, which was really used only by the container meta events. https://github.com/draios/sysdig/pull/912/commits/0e00ef67ff53293414fe2a1026910e19ab63fbc2 describes why in more detail.